lib/scripts/pdf_a4_portrait.js

```diff
@@ -153,11 +154,19 @@ function createSection (section, content, options) {
     height: options.height,
     contents: phantom.callback(function (pageNum, numPages) {
       var html = o[pageNum] || c[pageNum]
+
+      var pageNumFinal = pageNum, numPagesFinal = numPages;
```

Comment:
Please split up multiple assignments onto multiple lines.
This didn't throw an error in the linting script because only `bin/index.js` was tested.
I've created a pr here that fixes the config https://github.com/marcbachmann/node-html-pdf/pull/230/files

Can you please rebase and apply those changes.